### PR TITLE
Add bestselling section

### PR DESCRIPTION
**Pull request by paul-111**
Adding Best Seller section 

image

### CODE REVIEW

File: src/app/components/pages/Products/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { productDummyData } from "../../dummy/best-seller";
+
+export const Products = () => {
+  return (
+    <ContentWrapper>
+      <Typography variant="h2" sx={{ m: 10, textAlign: "center" }}>
+        Our bestsellers
+      </Typography>
+      <Container
+        style={{ display: "flex", flexDirection: "row", gap: 15 }}
```
Comment (mehedihassandev) [inline src/app/components/pages/Products/index.tsx:20]: use sx

**Review by mehedihassandev — CHANGES_REQUESTED** [review]
RC

File: src/assets/img/vase1.avif
Comment (sun-mit) [inline src/assets/img/vase1.avif:1]: why we use .avif ? I think svg/png good idea.

File: src/app/components/dummy/best-seller.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import vase1 from "../../../assets/img/vase1.avif";
+import vase2 from "../../../assets/img/vase2.avif";
+import vase3 from "../../../assets/img/vase3.avif";
+import vase4 from "../../../assets/img/vase4.avif";
+
+export const productDummyData = [
```
Comment (sun-mit) [inline src/app/components/dummy/best-seller.tsx:6]: good one to make dummy

File: src/assets/img/vase1.avif
Comment (paul-111) [reply src/assets/img/vase1.avif:1]: avif was original extension actually and didn't change once got saved, i can change it though, will update then

**Comment by mehedihassandev** [comment]
resolve conflicts ASAP @paul-111

**Comment by paul-111** [comment]
@mehedihassandev aii chup

**Review by mehedihassandev — APPROVED** [review]

**Pull request merged**